packages/web-components/fast-foundation/src/di/di.ts

```diff
@@ -303,7 +303,7 @@ export const DI = Object.freeze({
     },
 
     findResponsibleContainer(element: HTMLElement): Container {
-        const owned = (element as any).$container as ContainerImpl;
+        const owned = (element as any).$$container$$ as ContainerImpl;
```

Comment:
I'm curious what the impetus for this change is?

Reply:
There was someone saying that it was conflicting with their own properties on their elements. I just basically tried to make it less likely to conflict since we were just adding it on arbitrary nodes.

Reply:
gotcha okay cool